src/lib/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "noEmitHelpers": true,
+    "module": "es2015",
+    "moduleResolution": "node",
+    "target": "es2017",
```

Comment:
Nah, the ts should compile the latest js version possible because babel is applied right after that anyways.